### DISCUSSION TITLE
Remove ETH from Altcoins build

### DISCUSSION THREAD

**Discussion by britttttk** [btcpayserver/btcpayserver#3140] [General]
ETH doesn't seem to work and it's basically an abandoned altcoin that just wastes user's time and causes them frustration. I don't see any point to keeping it when users come to the chat asking about it and we simply tell them it's broken. 

https://chat.btcpayserver.org/btcpayserver/pl/95r78x9pbinwbfqh7ok4dbycaa

**Comment by astupidmoose** [comment]
Agreed, it's never worked. Until someone from the ETH community actually wants to create a working implementation I say this gets nixxed. At the very least the documentation should indicate that it's no longer working, or simply remove it.

**Comment by pavlenex** [comment]
NUKE IT
